AllReadyApp/Web-App/AllReady/Security/ClaimTypes.cs

```diff
@@ -24,5 +24,10 @@ public class ClaimTypes
         /// The user's local time zone
         /// </summary>
         public const string TimeZoneId = "ar:timezoneid";
+
+        /// <summary>
+        /// A claim indicating that the user's profile is incomplete
+        /// </summary>
+        public const string ProfileIncompleted = "ar:profileincomplete";

```

Comment:
"Incompleted" isn't really a word. :wink: Pretty sure you intended for it to be `ProfileIncomplete`.